CHANGES.md

```diff
@@ -1,3 +1,19 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured
+  * The `--proxy-auto-inject` flag  from the `install` command has been completely revamped
+  * The `inject` command proxy options are now converted into config annotations; the annotations ensure that these configs are persisted in subsequent resource updates
+  * Always show open TCP connections in the CLI
+  * The `stat` command now always shows the number of open TCP connections
+  * **Breaking Change** Removed the `--disable-external-profiles` flag from the `install` command; external profiles are now disabled by default and can be enabled with the new `--enable-external-profiles` flag
+* Controller
+  * The webhook config now reacts to changes in pods instead of deployments
+* Web UI
+  * The Octopus graph now shows TCP stats if no HTTP stats are available
+  * The resource detail page no longer shows blank tables if the resource only has TCP traffic
+  * Added validation to the new service profile form (thanks @liquidslr!)
```

Comment:
I would leave the quotation marks in there for clarity since the form is for creating new service profiles, it's not a new form:

`Added validation to the "new service profile" form`

Reply:
Ah that would explain the capital "N" in your original comment. Thanks!